gcovr/html_generator.py

```diff
@@ -88,11 +85,13 @@ def coverage_to_color(coverage):
 # Produce an HTML report
 #
 def print_html_report(covdata, options):
+    medium_threshold = options.html_medium_threshold
+    high_threshold = options.html_high_threshold
     details = options.html_details
     if options.output is None:
         details = False
     data = {}
-    data['HEAD'] = options.html_title
+    data['HEAD'] = options.title
```

Comment:
I think the tests are failing because of this line, otherwise this looks great!

Reply:
Already pushed. How do I start the test again?

Reply:
The tests restart automatically when you push. The remaining build failures are not test failures (wohoo!), but just style guide violations, see the build log: <https://travis-ci.org/gcovr/gcovr/jobs/392858479#L925>. To run the style checker locally, install flake8 from PyPI.

Reply:
Ok. At work I'm using SmartGit under Windows and at home I'm working with different tools under MacOs the first time.